pkg/testutil/dep.go

```diff
@@ -0,0 +1,79 @@
+package testutil
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/unpoller/unifi/mocks"
+	"github.com/unpoller/unpoller/pkg/inputunifi"
+	"github.com/unpoller/unpoller/pkg/poller"
+)
+
+func init() {
+	// async init
+
+	for i, arg := range os.Args[1:] {
+		if arg == "asyncWaitMultiplier" {
```

Comment:
What's this all about? Seems like an interesting way to do cli flags. Does this file get compiled into the app, or only used as a test? If compiled in, I'd rather see the init function not exist.

Reply:
didn't actually need this after re-factoring out the core poll logic into individual functions I can just invoke. Will remove this.